File: Konfigurator-aspnetcore-Backend/Konfigurator-aspnetcore-Backend/Models/DTOs/GetGameMapResponseDto.cs
using System;
namespace Konfigurator_aspnetcore_Backend.Models.DTOs
{
    public class GetGameMapResponseDto
    {
        public GetGameMapResponseDto(GetGameMapDto getGameMapDto)
        {
            this.getGameMapDto = getGameMapDto;
        }
        public GetGameMapDto getGameMapDto { get; }
    }
}

File: Konfigurator-nestjs-Backend/konfigurator-nestjs-backend/src/dto/get-GameMapResponse.dto.ts
import { ApiProperty} from '@nestjs/swagger';
import { GetGameMapDto } from "./get-GameMap.dto";

export class GetGameMapResponseDto {

    constructor(getGameMapDto: GetGameMapDto){
        this.getGameMapDto = getGameMapDto;
    }

    @ApiProperty()
    getGameMapDto: GetGameMapDto;
}